### PR TITLE
Fix indefinite installed page

### LINKED ISSUE #657: On systems with no Snaps installed, the Installed Page will load indefinitely.
There should be a "No Snaps installed" message coming up if no Snaps are installed. We could maybe do the same for the deb page? :grin:

https://user-images.githubusercontent.com/73116038/208285925-3dcdd2b9-1a48-4460-bcd4-28fbc08082fe.mp4

### PR DESCRIPTION
**Pull request by whiskeyPeak**
Doesn't use localisation until we figure out what kind of message we want to show. :D 

https://user-images.githubusercontent.com/73116038/211175419-871f268b-c7d4-46c2-b903-d92f6e937ec2.mp4

fixes #657

### CODE REVIEW

**Review by Feichtmeier — CHANGES_REQUESTED** [review]

**Comment by whiskeyPeak** [comment]
I was thinking that we could maybe rename it to "ResponsePage" and have it used wherever we should an empty page to the user with only an icon and message. This widget could be used:
- Explore page when the search can't find any correlated snaps/debs
- Installed page when no snaps are installed
- Installed page when no updates are available
- Update page when no updates are available

What do you think? @Feichtmeier

**Comment by Feichtmeier** [comment]
Reading through the code again and thinking about it, this is actually not an "error" so ErrorPage does not really fit here. It is just "No snaps are installed". please change the name of the page to something fitting and move it to the "snaps" directory. The message can be inside the en.arb file with something like

"noSnapsInstalled": "There are no snap applications installed on your system."

File: lib/app/explore/start_page.dart [hunk 1]
```diff
@@ -88,6 +89,9 @@ class _StartPageState extends State<StartPage> {
         controller: _controller,
         child: Column(
           children: const [
+            SizedBox(
+              height: kPagePadding - 5,
```
Comment (Feichtmeier) [inline lib/app/explore/start_page.dart:93]: whats that? :)

**Review by Feichtmeier — CHANGES_REQUESTED** [review]

File: lib/app/explore/start_page.dart (hunk 1 above)
Comment (Feichtmeier) [reply lib/app/explore/start_page.dart:93]: never mind, you merged master after and the diff was gone

**Review by Feichtmeier — APPROVED** [review]

File: lib/app/explore/start_page.dart (hunk 1 above)
Comment (whiskeyPeak) [reply lib/app/explore/start_page.dart]: Ahh yeah, I was wondering why it kept showing up lol

**Pull request merged**